.github/workflows/wpcs.yml

```
@@ -0,0 +1,26 @@
+name: WordPress Check Action And WPCS
+
+on: pull_request
+
+jobs:
+  phpcs:
+    name: Check Coding Standards
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Run plugin check
+        uses: swissspidy/wp-plugin-check-action@v1

```

Since you are already running PHPCS separetely below, there's little value in having Plugin Check running the same checks, so I'd recommend excluding those checks.

```suggestion
      - name: Run plugin check
        uses: swissspidy/wp-plugin-check-action@v1
          exclude-checks: |
              plugin_review_phpcs
```